easybuild/easyblocks/p/python.py

```diff
@@ -640,7 +638,11 @@ def make_module_extra(self, *args, **kwargs):
         """Add path to sitecustomize.py to $PYTHONPATH"""
         txt = super(EB_Python, self).make_module_extra()
 
+        # Legacy support for existing installations doing "--rebuild --module-only"
         if self.cfg.get('ebpythonprefixes'):
-            txt += self.module_generator.prepend_paths(PYTHONPATH, self.pythonpath)
+            new_dir = os.path.join('lib', 'python' + self.pyshortver, 'site-packages')
+            old_dir = os.path.join(log_path(), 'python')
```

Comment:
`log_path` isn't actually the correct path, see https://github.com/easybuilders/easybuild-easyblocks/pull/3400
Maybe we can either use `easybuild` directly (if I remember that path correctly) or at least add a comment why `log_path` is abused here

Reply:
I'm fine with either removing this whole block of code, since we have the "freedom" with EasyBuild 5.0 to introduce a breaking change like this in a very specific situation (`--rebuild --module-only` *across* major EasyBuild versions), or with hardcoding `'easybuild'` here instead of using `log_path()`...